lib/manager/components/transform/CustomCSVForm.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+
+import React from 'react'
+import { Button } from 'react-bootstrap'
+
+import { getComponentMessages } from '../../../common/util/config'
+
+type Props = {
+  csvData: ?string,
+  inputIsUnchanged: boolean,
+  onChangeCsvData: () => void,
+  onSaveCsvData: () => void,
+  type: string
+}
+const CustomCSVForm = (props: Props) => {
+  const { csvData, inputIsUnchanged, onChangeCsvData, onSaveCsvData, type } = props
+
+  const textValue = csvData || ''
+  const numLines = !textValue ? 0 : textValue.split(/\r*\n/).length
+  const messages = getComponentMessages('CustomCSVForm')
+
+  // Assign yaml messages based on component
+  let header, buttonText
+  if (type === 'ReplaceFileFromString') {
+    header = messages('addCsvData')
+    buttonText = messages('saveCsv')
+  } else if (type === 'AddCustomFile') {
+    header = messages('addCustomCsvData')
+    buttonText = messages('saveCsvAndFileName')
+  }
+
+  return <div>
+    <label
+      htmlFor='csvData'
+      style={{
+        display: 'flex',
+        justifyContent: 'space-evenly',
+        flexDirection: 'column'
+      }}
+    >
+      {header}
+      <textarea
+        id='csvData'
+        onChange={onChangeCsvData}
+        placeholder={
+          `stop_id,stop_code,stop_name,stop_lat,stop_lon\n1234567,188390987,Broad Ave,33.98768,-87.72686`
```

Comment:
This is a template string, you shouldn't need `\n`

Reply:
I added the \n just so it looks better in the code, otherwise we need to put it on a newline and can't tab it over. Lmk what you think it was a superficial change anyway.